docs/datasets_and_tables/gtfs_schedule.md

```diff
@@ -1,15 +1,44 @@
-# GTFS Schedule (WIP)
+# GTFS Schedule
 
-## Data
+## How to view the `GTFS Schedule` documentation
 
-| dataset | description |
+Visit the [dbt Cal-ITP warehouse documentation](https://dbt-docs.calitp.org/#!/overview).
+
+For help navgating the documentation, visit [Navigating the dbt Docs](navigating-dbt-docs).
+
+## Warehouse Schemas
+
+### Source
+
+| dataset name | description |
 | ------- | ----------- |
-| gtfs_schedule | Latest warehouse data for GTFS Static feeds. See the [GTFS static reference](https://developers.google.com/transit/gtfs/reference). |
-| gtfs_schedule_type2 | Tables with GTFS Static feeds across history (going back to April 15th, 2021). These are stored as type 2 slowly changing dimensions. They have `calitp_extracted_at`, and `calitp_deleted_at` fields. |
-| (internal) gtfs_schedule_history | External tables with all new feed data across history. |
+| `gtfs_schedule_type2` | Tables with GTFS-Static feeds across history (going back to April 15 2021). These are stored as type 2 slowly changing dimensions. They have `calitp_extracted_at` and `calitp_deleted_at` fields. |
+| `gtfs_schedule_history` | Data in the `gtfs_schedule_history` dataset in BigQuery |
+
+### Staging
+
+| dataset name | description |
+| ------- | ----------- |
+| `gtfs_schedule` | Latest warehouse data for GTFS-Schedule feeds. |
+
+### See Also
+The [GTFS-Schedule Data Standard](https://developers.google.com/transit/gtfs/).
+
+## `dbt` Project Directories
+More information on the [navigating the docs from the project-perspective](navigating-dbt-docs)
 
-## Dashboards
+### Source tables
+| Directory name |
+| ------- |
+| `gtfs_schedule_history` |
+| `gtfs_type2` |
 
-## DAGs Maintenance
+### Staging tables
+| Directory name |
+| ------- |
+| `gtfs_schedule_latest_only` |
 
-You can find further information on DAGs maintenance for GTFS Schedule data [on this page](gtfs-schedule-dags).
+:::{admonition} See Also: `views` documentation
+:class: tip
```

Comment:
I think the little `tip` style is so cute 🥺

Reply:
I probably use it unnecessarily because I think it's so cute